### DISCUSSION TITLE
第1节课笔记(10班)

### DISCUSSION THREAD

**Discussion by isLinXu** [InternLM/Tutorial#58] [笔记]
第一节课笔记（10班）👇

**Comment by zhangyutuan** [comment]
https://www.processon.com/view/link/65956b822a264f3868996636

**Comment by Yuan1z0825** [comment]
https://blog.csdn.net/stn54999/article/details/135374291?spm=1001.2014.3001.5502

**Comment by jiangye520** [comment]
https://mp.weixin.qq.com/s/MuLtcSRntU7MQSW3D09-9Q

**Comment by MIracleyin** [comment]
https://nnkwjqimr7.feishu.cn/docx/BlnOd3u4soGLcgxVZkZcnHAtnnf

**Comment by dearSocrates** [comment]
https://blog.csdn.net/rabbit9798/article/details/135375478?spm=1001.2014.3001.5502

**Comment by lishuang70345** [comment]
https://blog.csdn.net/weixin_42748604/article/details/135379858?spm=1001.2014.3001.5502

**Comment by WalkAndLook** [comment]
https://blog.csdn.net/qq_43494330/article/details/135391378

**Comment by admlrz** [comment]
https://blog.csdn.net/weixin_44581312/article/details/135392952?spm=1001.2014.3001.5501

**Reply by MIracleyin** [reply]
同学，请更新一下你的 GitHub ID，这边表格找不到你的名字

**Comment by cemcke** [comment]
http://t.csdnimg.cn/lulY7

**Comment by aerial1991** [comment]
https://blog.csdn.net/aerial1991/article/details/135400474?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135400474%22%2C%22source%22%3A%22aerial1991%22%7D

**Comment by Tian-CASIA** [comment]
https://blog.csdn.net/qq_55090315/article/details/135407949?spm=1001.2014.3001.5502

**Comment by hepingpeace** [comment]
http://t.csdnimg.cn/Bffsi

**Comment by jingwanli6666** [comment]
https://blog.csdn.net/jasonjwl/article/details/135391538?spm=1001.2014.3001.5502

**Comment by MESSIwy** [comment]
https://zhuanlan.zhihu.com/p/676472847

**Comment by hannah803** [comment]
https://hannahliu.framer.website/blog/internlm-training-lec-1

**Comment by adela778** [comment]
https://blog.csdn.net/Adela0546/article/details/135417333

**Comment by lvcaiwen** [comment]
https://bbs.csdn.net/topics/617879944

**Reply by MIracleyin** [reply]
同学，请更新一下你的 GitHub ID，这边表格找不到你的名字

**Comment by lishuang70345** [comment]
***@***.***


从 Windows 版邮件发送

发件人: Yin Zhang
发送时间: 2024年1月11日 0:28
收件人: InternLM/tutorial
抄送: aaaashuang; Comment
主题: Re: [InternLM/tutorial] 第1节课笔记(10班) (Discussion #58)

同学，请更新一下你的 GitHub ID，这边表格找不到你的名字

**Comment by Tian-CASIA** [comment]
抱歉给您造成麻烦，请问是在书生·浦语大模型实战营10班信息表 - 飞书云文档 (feishu.cn)文档中更新github id吗？github id到底是需要填什么内容，我之前填的是注册邮箱不可以吗？是需要换成用户名，在github设置里没有找到id项。多有麻烦，不胜感激！


祝好




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "InternLM/tutorial"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月11日(星期四) 上午8:50
***@***.***&gt;;
抄送:&nbsp;"感恩  ***@***.******@***.***&gt;;
主题:&nbsp;Re: [InternLM/tutorial] 第1节课笔记(10班) (Discussion #58)





 ***@***.*** 
 
 
 从 Windows 版邮件发送 
 
 发件人: Yin Zhang 
 发送时间: 2024年1月11日 0:28 
 收件人: InternLM/tutorial 
 抄送: aaaashuang; Comment 
 主题: Re: [InternLM/tutorial] 第1节课笔记(10班) (Discussion #58) 
 
 同学，请更新一下你的 GitHub ID，这边表格找不到你的名字
